src/inausoft.netCLI.Tests/RegexArgumentDeserializerTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using inausoft.netCLI.Deserialization;
+using inausoft.netCLI.Tests.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace inausoft.netCLI.Tests
+{
+    [TestClass]
+    public class RegexArgumentDeserializerTests
+    {
+        [TestMethod]
+        public void RootCommandHandler_RunsProperCommandHandler_ForMultipleOptions()
+        {
+            //Arrange
+            var stringOptionValue = "sampleString";
+            var intOptionValue = 102;
+
+            var args = new string[]
```

Comment:
I think that the `arguments` and `options` (as in `OptionAttribute` are both used to denote the same thing - argument s for the command. Is it a possiblity to unify the naming? (I would suggest change `OptionAttribute` to `ArgumentAttribute`)

Reply:
`--option optionValue` are normaly called option hence the naming. I'm open to adjust it. What do you think @xHoody ?